packages/containers/.storybook/config.js

```diff
@@ -81,22 +83,51 @@ function selectTab(event, data) {
 	};
 }
 
+function httpPhotosGet1() {
+	return actions.http.get('https://jsonplaceholder.typicode.com/photos/', {
+		cmf: {
+			collectionId: 'photos1',
+		},
+	});
+}
+function httpPhotosGet2() {
+	return actions.http.get('https://jsonplaceholder.typicode.com/photos/', {
+		cmf: {
+			collectionId: 'photos2',
+		},
+	});
+}
+
 const registerActionCreator = api.actionCreator.register;
 registerActionCreator('object:view', objectView);
 registerActionCreator('cancel:hide:dialog', hideDialog);
 registerActionCreator('confirm:dialog', confirmDialog);
 registerActionCreator('item1:action', chooseItem1);
 registerActionCreator('item2:action', chooseItem2);
 
-registerActionCreator('subheaderbar:display-sharing', actionsCreatorsSubHeader.sharingSubHeader);
-registerActionCreator('subheaderbar:display-bubbles', actionsCreatorsSubHeader.bubblesSubHeader);
-registerActionCreator('subheaderbar:submit', actionsCreatorsSubHeader.submitSubheader);
-registerActionCreator('subheaderbar:edit', actionsCreatorsSubHeader.editSubHeaderBar);
-registerActionCreator('subheaderbar:cancel', actionsCreatorsSubHeader.cancelSubHeaderBar);
-registerActionCreator('subheaderbar:change', actionsCreatorsSubHeader.changeSubHeaderBar);
-registerActionCreator('subheaderbar:goback', actionsCreatorsSubHeader.goBackSubHeaderBar);
 
-registerActionCreator('tabbar:select', selectTab);
+api.actionCreator.register('http:get:photos1', httpPhotosGet1);
+api.actionCreator.register('http:get:photos2', httpPhotosGet2);
+api.actionCreator.register('object:view', objectView);
+api.actionCreator.register('cancel:hide:dialog', hideDialog);
+api.actionCreator.register('confirm:dialog', confirmDialog);
+api.actionCreator.register('item1:action', chooseItem1);
+api.actionCreator.register('item2:action', chooseItem2);
```

Comment:
You register the 5 previous action creators, but they are already registered above.
Plus, you `aoi.actionCreator.register` but it is extarcted into `registerActionCreator` var. Please align all the register calls.